modules/db/src/test/resources/integration/template/template-config.xml

```diff
@@ -57,6 +57,7 @@
 
     <db:template-query name="testNullParamsQuery">
         <db:parameterized-query>SELECT * FROM PLANET WHERE POSITION = :position</db:parameterized-query>
+        <db:in-param name="position" defaultValue="null"/>
```

Comment:
In a previous comment I mentioned that if you are going to use  "NULL" to set a null value then you hsould also support setting "NULL" when a template is used on a DB message processor (or when the query is defined implicitly inside the message processor). Please add another test to cover those scenarios.